code/datums/emergency_calls/upp.dm

```diff
@@ -1,16 +1,17 @@
 /datum/emergency_call/upp
-	name = "UPP"
-	probability = 10
+	name = "USL"
+	probability = 0
```

Comment:
This means the emergency call won't happen anymore by default. Only through admin intervention.